changes/202202151234.feature

```diff
@@ -0,0 +1 @@
+Possibility to create `OPTIONS` requests using an http.IClient
```

Comment:
[According to the way news files should be made](https://github.com/ARM-software/golang-utils/blob/dc840f91f3436c712f2d7355dc2015759db4b649/CONTRIBUTING.md#news-files), they should be YYYYMMDD not YYYYMMDDhhmm and you only need one for the whole merge, I see why you did this and it makes sense but then should the news files notes be updated?